### DISCUSSION TITLE
Welcome to py-libp2p Discussions!

### DISCUSSION THREAD

**Discussion by dhuseby** [libp2p/py-libp2p#439] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pacrob** [comment]
Hello! I was added as a maintainer to help out some months ago, but I haven't heard anything since. I wasn't sure if the plan was still to bring this lib up to speed, but this Welcome is some action! I don't have any previous experience directly with `libp2p`, but happy to learn and help where I can.

**Comment by dhuseby** [comment]
@pacrob I just emailed you about maintenance of py-libp2p.

**Reply by dhuseby** [reply]
Test reply here.

**Reply by dhuseby** [reply]
and another one.

**Comment by dhuseby** [comment]
with one more reply down here
